src/components/site-form.tsx

```diff
@@ -35,6 +35,20 @@ interface SiteFormErrorProps {
 	className?: string;
 }
 
+/**
+ * Generates a suitable domain name from site name
+ */
+const generateDomainFromSiteName = ( siteName: string ): string => {
+	// Convert the site name to lowercase and replace spaces with hyphens
+	const domainBase = siteName
```

Comment:
The current implementation doesn't work for languages that use characters that don't transliterate to a-z ASCII chars. Could we try reusing a logic from the existing `sanitizeFolderName()` function, which currently supports all supported languages?